lib/kb_uploadmethods/Utils/ImportSRAUtil.py

```diff
@@ -57,6 +58,14 @@ def _sra_to_fastq(self, scratch_sra_file_path, params):
         tmp_dir = os.path.join(self.scratch, str(uuid.uuid4()))
         handler_utils._mkdir_p(tmp_dir)
 
+        # PTV-1356 fix:  to try to make handing of output files from fastq-dump
+        # more uniform for cases where there is an SRA extension or not
+
+        # force an .sra extension if there isn't one already:
+
+        if not re.match( ".*\.sra$", scratch_sra_file_path, re.IGNORECASE ):
+            scratch_sra_file_path += ".sra"
```

Comment:
Did your test pass? I will be surprised if they pass. You didn’t rename the file. But instead you changed scratch_sra_file_path string. Now whenever you try to read scratch_sra_file_path, it will throw file not found error. Because the real file is not changed.

Reply:
I think you should completely revert SRAUtil since file name issue is fixed already.

Reply:
I *thought* I did that.  I spent an hour fighting with git to do just that, but it kept claiming there was a conflict, so I manually edited out what it marked, but I didn't notice it kept some of my code